src/Widgets/MprisGui.vala

```diff
@@ -58,13 +58,10 @@ public class Sound.Widgets.ClientWidget : Gtk.Grid {
                 if (app_name == "") {
                     app_name = ainfo.get_name ();
                 }
-
-                app_icon = value.get_icon ();
-                if (app_icon == null) {
-                    app_icon = new ThemedIcon ("application-default-icon");
+                if (value.get_icon () != null) {
+                    app_icon = value.get_icon ();
```

Comment:
It's probably still a good idea to cache the value of `value.get_icon ()` here since we use it twice in a row, but other than that this seems sane to me :+1: